File: api/python/examples/consumer.py
#encoding=utf8  
from zbus import SingleBroker, Consumer

broker = SingleBroker(host='localhost', port=15555)

consumer = Consumer(broker=broker, mq='MyMQ')  
while True:
    msg = consumer.recv()
    if msg is None: continue
    print msg
    
    

File: api/nodejs/examples/consumer.js
var zbus = require("../zbus");
var Message = zbus.Message;
var MessageClient = zbus.MessageClient;
var Consumer = zbus.Consumer;


var client = new MessageClient("127.0.0.1:15555");
client.connect(function(){
    var c = new Consumer(client, "MyMQ");
    c.recv(function(msg){
        console.log(msg.toString());
    });
});
